olad/www/new/index.html

```diff
@@ -54,6 +54,15 @@
  </div>
  <!-- /.container-fluid -->
 </nav>
+<noscript>
+  <br>
+  <p align="center">
+    <strong>This interface requires JavaScript to function. If you
```

Comment:
Right you are, in copying and pasting, the indentation got messed up. Fixing now.